src/main/java/count/Count.java

```diff
@@ -0,0 +1,85 @@
+package count;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import count.action.Action;
+import count.action.Deactivate;
+import count.exception.CountException;
+import count.exception.IncorrectFormatException;
+
+/**
+ * Count is a lightweight helper with an inbuilt to-do list
+ * @author Kieran Koh Jun Wei
+ */
+public class Count {
+    private TaskList ls;
+    private boolean isOn;
+    private Scanner sc;
+    private UI ui;
+    private Storage storage;
+    private Parser parser;
+
+    /**
+     * Constructor for Count object
+     * @param filePath String filePath in which Count will read and save to
+     */
```

Comment:
- In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... etc. (not Return or Returning etc)
- Space between method summary and list of parameters (not explicitly required but shown in example given in java coding standards page, and also looks nice)
```suggestion
    /**
     * Constructs Count object
     *
     * @param filePath String filePath in which Count will read and save to
     */
```